src/sentry/integrations/slack/client.py

```diff
@@ -38,7 +38,9 @@ def __init__(
                 integration_id=self.integration_id, ctx_logger=logger
             )
 
-        super().__init__(org_integration_id, verify_ssl, logging_context)
+        super().__init__(
+            org_integration_id, verify_ssl, integration_id, logging_context=logging_context
```

Comment:
These args are out of order - check out the order the `IntegrationProxyClient`'s init expects them: https://github.com/getsentry/sentry/blob/54a6bdacdcdba8399d2a09f8d162d4d886ee3578/src/sentry/shared_integrations/client/proxy.py#L69-L72

you can either name them e.g.

```
        super().__init__(
            org_integration_id=org_integration_id, verify_ssl=verify_ssl, integration_id=integration_id, logging_context=logging_context
        )
```

or change the order they're passed in e.g.:

```
        super().__init__(
            integration_id, org_integration_id, verify_ssl, logging_context
        )
```